### PR TITLE
PIM-5643 : Fix default system locale was not fixed by the last PIM-5643 ticket, tagged in 1.5.2

### PR DESCRIPTION
**Pull request by momoss**
Backport of this fix https://github.com/akeneo/pim-community-dev/commit/5d326e2ea654e5bba955e7e776c0853c267863f4

### CODE REVIEW

File: CHANGELOG-1.5.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 - Upgrade "akeneo/measure-bundle" from "0.4.1" to "0.5.0", details in the release note https://github.com/akeneo/MeasureBundle/releases/tag/0.5.0
 
+## Bug fixes
+
+- PIM-5643: Fix default system locale was not fixed by the last PIM-5643 tagged in 1.5.2
```
Comment (phaseinducer) [inline CHANGELOG-1.5.md]: Something is missing in this sentence

**Comment by skeleton** [comment]
commit should be a cherry-pick to avoid problem when 1.5 will be merge into master

**Comment by momoss** [comment]
Cherry-pick well done, and update changelog message @phaseinducer

**Comment by skeleton** [comment]
GTM

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**